src/alpine/3.6/Dockerfile

```diff
@@ -26,7 +26,10 @@ RUN apk add --no-cache \
         paxctl \
         python \
         util-linux-dev \
-        zlib-dev
```

Comment:
Please alphabetize packages.  It helps the readability and is a [Dockerfile best practice](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#sort-multi-line-arguments).